pkg/appmanager/types.go

```diff
@@ -57,52 +61,79 @@ type (
 		VirtualServerName string `json:"name"`
 		PoolName          string `json:"pool,omitempty"`
 		// Mutual parameter, partition
-		Partition string `json:"partition"`
+		Partition string `json:"partition,omitempty"`
 
 		// VirtualServer parameters
-		Balance        string          `json:"balance,omitempty"`
-		Mode           string          `json:"mode,omitempty"`
-		VirtualAddress *virtualAddress `json:"virtualAddress,omitempty"`
-		SslProfile     *sslProfile     `json:"sslProfile,omitempty"`
-		Policies       []nameRef       `json:"policies,omitempty"`
-		IRules         []string        `json:"rules,omitempty"`
-		// FIXME: All profiles should reside in Profiles, just server ssl ones now.
```

Comment:
This comment shouldn't be removed since SslProfile is still here (we have profiles in more than one spot still).

Reply:
sslProfile just contains the profile names parsed from the ConfigMap. I convert each of these to a ProfileRef and put them in Profiles, which gets shipped to CCCL.

Reply:
Yes, but our goal is to get the go code to eventually be closer to the CCCL layout and not the configmap + CCCL layouts combined, so in that scenario all profiles would go into Profiles and all the quirky configmap stuff would be translated when configmaps are parsed. So the comment is still valid for the next step.

Reply:
OK, I will keep the comment.